index.html

```diff
@@ -0,0 +1,331 @@
+    </section>
+    <section class="works-card">
+      <img class="snapshort" src="./images/snap4.png" alt="snapshoot" />
+      <div class="left-block">
+        <div class="first-container">
+          <h2 class="newme-title">
+            Multi-Post Stories<br />
+          </h2>
+          <div class="event2-tonic">
+            <div class="word">
+              <h3>CANOPY</h3>
+            </div>
+            <img src="./images/Counter.png" alt="counter count" />
+            <div class="backend">
+              Back End Dev
+            </div>
+            <img src="./images/Counter.png" alt="counter count" class="dot" />
+            <div class="backend">
+              2015
+            </div>
+          </div>
+        </div>
+        <p class="newme-description-tonic">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li class="language1">
+            html
+          </li>
+          <li class="language2">
+            css
+          </li>
+          <li class="language3">
+            javascript
+          </li>
+        </ul>
+      </div>
+    </section>
+      <div class="newmes">
+        <button type="submit" class="go-newmes">See projects</button>
+      </div>
+    </div>
+  </section>
+</main>
+  <section class="contact-me">
+    <div class="head-section-co">
+      <div id="head-container-co">
+        <h2 class="head-co">
+          About <br />
+          Myself
+        </h2>
+        <p class="description-co">
+          Hello I’m a software developer! I can help you build a product,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <h2 class="connect-co">LET’S CONNECT</h2>
+
+        <ul class="social-media-icons">
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/github.png" alt="git" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/Linkedin.png" alt="linkedIn" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/angelite.png" alt="image" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/twitter.png" alt="image" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/medium.png" alt="image" />
+            </a>
+          </li>
+        </ul>
+        <div class="new-code">
+          <button type="submit" class="go-newmes">My resume</button>
+        </div>
+
+        <div class="skill">
+          <div class="event12">
+            <div class="event7">
+              <h3 class="languages">Languages</h3>
```

Comment:
- [ ] As  suggestted by the previous reviewer please use`<ul>`and`<li>` semantic tags for the Framworks and skills. So that all the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul` not in a `<div>`, each of them in a li and the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`

> 
        <ul>
    <li><h3>Languages</h3>
      <ul>
        <li>Javascript</li>
        <li> HTML </li>
        <li>CSS</li>
      </ul>
    </li>
    <li><h3>Frameworks</h3></li>
    <li><h3>Skills</h3></li>
        </ul>

Reply:
Good morning!!

I made all changes recommended before. This is my last extra review. Please, check it ASAP.

        <ul class="skill">
          <li class="event12">
            <li class="event7">
              <h3 class="languages">Languages</h3>
              <button type="submit" class="dropdown">
                <img src="./images/Icon-Chevron-Down-Larger.png" alt="" />
              </button>
            </li>            
              <li class="language-list">
                <ul>
                  <li class="JavaScript1">
                    <div class="lang">
                      <img src="./images/Ellipse-1.png" alt="" />
                      <p>JavaSript</p>
                    </div>
                  </li>
                  <li class="HTML1">
                    <div class="lang">
                      <img src="./images/Ellipse-2.png" alt="" />
                      <p>HTML</p>
                    </div>
                  </li>
                  <li class="CSS1">
                    <div class="lang">
                      <img src="./images/Ellipse-3.png" alt="" />
                      <p>CSS</p>
                    </div>
                  </li>
                </ul>
              </li>	            
          </li>
          <ul class="event-skill">
            <li class="event18">
              <h3>Frameworks</h3>
              <button type="submit" class="dropdown">
                <img src="./images/Disabled.png" alt="" />
              </button>
            </li>
            <img src="./images/Separator-Bottom.png" alt="" class="separator" />
            <li class="event18">
              <h3>Skills</h3>
              <button type="submit" class="dropdown">
                <img src="./images/Disabled.png" alt="" />
              </button>
            </li>
            <img src="./images/Separator-Bottom.png" alt="" class="separator" />
          </ul>
        </ul>